docs/api-versions.md

```diff
@@ -0,0 +1,12 @@
+# API versions
+
+## Supported API Versions
+
+The Performance Addon Operator supports *v1* and *v1alpha1* [Performance Profile](docs/performance_profile.md) versions.
+There are no differences between the *v1* and *v1alpha1* versions, the upgrade has been made in order to
+mark the Performance Profile API as stable.
+
+## Upgrade from *v1alpha1* to *v1*.
+When upgrading from an older Performance Addon Operator version supporting only *v1alpha1* API to a newer one supporting
+also the *v1* API, the existing *v1alpha1* Performance Profiles will be converted on-the-fly using a "None" Conversion
+strategy and served to the Performance Addon Operator as *v1*.
```

Comment:
what happens in practice if I install a v1-enabled PAO on my cluster? what should I expect?

Reply:
PAO will expect v1 Performance Profiles and query only them. Existing v1alpha1 profiles are served as v1 and PAO works with them as usual

Reply:
yep, and these flows should explained in this doc, these are the examples people will look up for :)

Reply:
Also, worth pointing out that there is a strong overlap (and that's good!) between the flows we document here and the flow we test in https://github.com/openshift-kni/performance-addon-operators/pull/392/commits/fca81fd9940925befa0f944c2baf7464e7737520

Reply:
Added a Q&A section with the above. Regarding the commit you mentioned - I will update the doc as soon as the Conversion Webhook will be merged.

Reply:
Not sure I understood completely: we should be able to check before we merge that the webhook e2e tests cover the flow we document. Or do we have reasons I'm not seeing to have the two PRs diverge?

Reply:
These are not related PRs. This PR is to be applied on the current master and does not depend on the Webhook Conversion PR.
This PR covers the state "before" the webhook conversion PR

Reply:
I plan to add on top of this doc explanations as part of V2 API PR. I added this PR as an out-of-band one since I have a JIRA issue for this sprint. It just covers current state: v1alha1 <-> v1 with None conversion strategy.

Reply:
Yes, the two PRs obviously don't depend on each other, but they are both about the same topic, that's what I meant

Reply:
yes, they are, this is why it will need a re-spin soon, is on my todo list.